Libraries/Components/MapView/MapView.js

```diff
@@ -35,6 +35,14 @@ type MapRegion = {
   longitudeDelta: number;
 };
 
+var ANNOTATION_DRAG_STATE = [
+  'Idle',
+  'Starting',
+  'Dragging',
+  'Canceling',
```

Comment:
The preferred way to handle enums like this in RN is with an RCTConvert category that maps lower-case string names to the underlying native enum automatically, and a flow $Enum<> in the JS.

See AlertIOS/RCTAlertManager for an example.